RoboJackets/igvc-software
# Feat/scan to pointcloud rewrite

# Description

Refactors the scan_to_pointcloud node into a header and cpp file

Fixes #741 

# Testing steps (If relevant)
1. Run a simulation in master branch
2. Run a simulation in feat/scan_to_pointcloud_rewrite branch
3. Check that the scan_to_pointcloud node functions the same way in both simulations

Expectation: The scan_to_pointcloud node should maintain functionality

# Self Checklist
